src/pseudopeople/utilities.py

```diff
@@ -7,22 +7,25 @@
 from vivarium.framework.randomness import RandomnessStream
 from vivarium.framework.randomness.index_map import IndexMap
 
-from pseudopeople.constants import metadata, paths
+from pseudopeople.constants import metadata
 
 
-def get_randomness_stream(dataset_name: str, seed: int) -> RandomnessStream:
+def get_randomness_stream(dataset_name: str, seed: int, index: pd.Index) -> RandomnessStream:
+    map_size = max(1_000_000, max(index)*2)
```

Comment:
Len of index?

Reply:
No, the actual index maximum. If we pass in a dataset with, say, and index [1,100], then len(index)=2 but we would try and assign something to loc[100].